src/pages/MenteeRegistration/index.tsx

```diff
@@ -126,6 +126,9 @@ const MenteeRegistration: React.FC = () => {
         { withCredentials: true }
       );
     },
+    onSuccess: async () => {
+      await refetch();
```

Comment:
Let's invalidate the query instead of  refetch

Reply:
That's what `refetch` method does, isn't? It was written in `useProfile`.

Reply:
oh yes, my bad

Reply:
No worries 🙌🏼